WORKSPACE

```diff
@@ -1,59 +1,22 @@
 workspace(name = "com_google_jsinterop_generator")
 
-maven_server(
-    name = "sonatype_snapshot",
-    url = "https://oss.sonatype.org/content/repositories/snapshots",
-)
-
-maven_jar(
-    name = "jsr305",
-    artifact = "com.google.code.findbugs:jsr305:3.0.1",
-)
-
-maven_jar(
-    name = "guava",
-    artifact = "com.google.guava:guava:21.0",
-)
-
-maven_jar(
-    name = "args4j",
-    artifact = "args4j:args4j:2.33",
-)
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-maven_jar(
-    name = "auto_value",
-    artifact = "com.google.auto.value:auto-value:1.4",
+http_archive(
+    name = "google_bazel_common",
```

Comment:
I don't think we should use that. It does not seens to be an official project from bazel team.